target/linux/bmips/bcm6328/config-6.6

```diff
@@ -208,6 +209,7 @@ CONFIG_PADATA=y
 CONFIG_PAGE_POOL=y
 CONFIG_PAGE_SIZE_LESS_THAN_256KB=y
 CONFIG_PAGE_SIZE_LESS_THAN_64KB=y
+CONFIG_PAHOLE_HAS_LANG_EXCLUDE=y
```

Comment:
Drop these, I pushed a filter update so these dont end up in configs:
https://github.com/openwrt/openwrt/commit/d02071c24f505ddba74b52c4ccd590e44ca8977d